### PR TITLE
♻️ PG page improvements

### PR DESCRIPTION
**Pull request by AuroraHuang22**
- Remove the plyr package, and use iframe and CSS for RWD.
- Improve the readability of the classes.
- Extract reusable components.

### CODE REVIEW

File: src/pages/gutenberg/index.vue [hunk 1]
```diff
@@ -256,59 +376,106 @@
...
-                  src="https://www.youtube.com/embed/85DOlacoy5Q"
-                  title="What is Decentralized Publishing"
-                  frameborder="0"
-                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
-                  allowfullscreen
-                ></iframe>
-              </div>
-            </VuePlyr>
+          <!-- 16:9 container -->
+          <div :class="['relative', 'w-full', 'h-0', 'pb-[56.25%]']">
```
Comment (nwingt) [inline src/pages/gutenberg/index.vue:410]: Can use [`aspect-video`](https://tailwindcss.com/docs/aspect-ratio) ?

File: src/pages/gutenberg/index.vue [hunk 2]
```diff
@@ -149,104 +326,47 @@
...
-                @click="() => handleClickDownload(book.classId)"
-              />
-            </div>
-          </div>
+        <div v-for="book of freeDownloadList" :key="book.classId">
+          <GutenbergDownloadItem
+            :book-info="book"
+            @download="handleClickDownload"
+          />
         </div>
```
Comment (nwingt) [inline src/pages/gutenberg/index.vue:349]: Please also update `<GutenbergDownloadItem/>` root compoent to `<li/>` or support `tag="li"` props
```suggestion
        <ul v-for="book of freeDownloadList" :key="book.classId">
          <GutenbergDownloadItem
            :book-info="book"
            @download="handleClickDownload"
          />
        </ul>
```

File: src/pages/gutenberg/index.vue (hunk 1 above)
Comment (AuroraHuang22) [reply src/pages/gutenberg/index.vue:410]: 🫨🫨🫨🫨🫨 我活到現在才知道有這個，真是不應該

Comment (AuroraHuang22) [reply src/pages/gutenberg/index.vue]: [589ca61](https://github.com/likecoin/liker-land/pull/1468/commits/589ca61bff8fc66b8569756ec592e1d027ed3a24)

File: src/pages/gutenberg/index.vue (hunk 2 above)
Comment (AuroraHuang22) [reply src/pages/gutenberg/index.vue]: [281b57a](https://github.com/likecoin/liker-land/pull/1468/commits/281b57abd17ba843aaa0e4767a17175896f5d336)

**Review by williamchong — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**